kitsune/sumo/static/sumo/scss/base/forms/_button-wrap.scss

```diff
@@ -38,6 +38,11 @@
   margin-bottom: -#{p.$spacing-sm};
   padding: p.$spacing-sm 0;
   width: 100%;
+  max-width: 95%;  // Default for mobile
```

Comment:
Please remove redundant comments. This comment applies to all the changed files

Reply:
Got it.